site/content/docs/5.0/forms/validation.md

```diff
@@ -351,14 +351,16 @@ If your form layout allows it, you can swap the `.{valid|invalid}-feedback` clas
 
 ## Customizing
 
-Validation states can be customized via Sass with the `$form-validation-states` map. Located in our `_variables.scss` file, this Sass map is looped over to generate the default `valid`/`invalid` validation states. Included is a nested map for customizing each state's color and icon. While no other states are supported by browsers, those using custom styles can easily add more complex form feedback.
+Validation states can be customized via Sass with the `$form-validation-states` map. Located in our `_variables.scss` file, this Sass map is how we generate the default `valid`/`invalid` validation states. Included is a nested map for customizing each state's color, icon, tooltip color, and focus shadow. While no other states are supported by browsers, those using custom styles can easily add more complex form feedback.
 
 Please note that we do not recommend customizing these values without also modifying the `form-validation-state` mixin.
 
 This is the Sass map from `_variables.scss`. Override this and recompile your Sass to generate different states:
 
 {{< scss-docs name="form-validation-states" file="scss/_variables.scss" >}}
 
+Maps of `$form-validation-states` can contain three optional parameters to override for tooltips and focus styles.  Not setting an override will use the default values.
```

Comment:
Two spaces before "Not".

Also do we need to mention the default values here? We don't anywhere else, , AFAIK.